### ISSUE TITLE
Collections counter not updated

### ISSUE THREAD

**Issue by alfredotoselli** [cheshire-cat-ai/admin-vue#110]
After chatting with the cat and navigating to Memory view the counters do not reflect the current state.

The only way to update counters is to refresh the page.

Could be confusing for newbies

**Comment by zAlweNy26** [comment]
Yeah it's already known, it was "not done" so that it would not spam requests to the backend each time visiting the memory page. Do you have any idea?

**Comment by alfredotoselli** [comment]
Ahhh ok just opened ahah
I'll close it, no idea for the moment

**Comment by zAlweNy26** [comment]
For me it's ok, I just don't know if it's the right choice ahah

**Comment by zAlweNy26** [comment]
Maybe a refresh button could be added

**Comment by alfredotoselli** [comment]
I thought it was not a big deal for an admin interface used at most by a couple of user, for me is something mergable.

Another could be to update the counters when the user does the search, maybe with an info that says the counter is not updated until search is performed

**Comment by zAlweNy26** [comment]
@kodaline @sambarza what do you think?

**Comment by sambarza** [comment]
> Maybe a refresh button could be added

I agree, low priority for my side

**Comment by kodaline** [comment]
> @kodaline @sambarza what do you think?

A refresh button might be a solution, I see this as a very low priority though, it's not an issue, but an implementation choice after all
